overlord/hookstate/ctlcmd/ctlcmd.go

```diff
@@ -91,19 +91,44 @@ func addCommand(name, shortHelp, longHelp string, generator func() command) {
 	}
 }
 
+type ForbiddenCommand struct {
+	Uid     uint32
+	Command string
```

Comment:
This should likely be Name instead (the command is the ForbiddenCommand instance).

Reply:
Done.